webhook-pubsub-publisher/src/app.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2021 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as dotenv from 'dotenv';
+import {PubSub} from '@google-cloud/pubsub';
+import type {Context, Probot} from 'probot';
+
+export default (app: Probot): void => {
+  if (process.env.NODE_ENV !== 'test') {
+    dotenv.config();
+  }
+
+  app.onAny(async (context: Context): Promise<void> => {
+    context.log(`Received "${context.name}" event with ID ${context.id}`);
```

Comment:
Which webhooks do we care about? Is it worth filtering here

Reply:
Doesn't matter! I'm keeping this open so that it can forward everything to the pub/sub topic - the subscribers can do the filtering directly via the subscription or in the application itself. Keeping this open to everything would let us pivot quickly if we ever decide to reuse the same pub/sub topic with other subscribers!